--- /home/jsirois/.cache/pants/setup/bootstrap-Linux-x86_64/0.0.81/lib/python2.7/site-packages/pants/ivy/bootstrapper.py.orig	2016-04-10 19:49:17.042975324 -0600
+++ /home/jsirois/.cache/pants/setup/bootstrap-Linux-x86_64/0.0.81/lib/python2.7/site-packages/pants/ivy/bootstrapper.py	2016-04-10 19:49:38.768525253 -0600
@@ -156,6 +156,5 @@
           raise self.Error('Problem fetching the ivy bootstrap jar! {}'.format(e))
 
     return Ivy(bootstrap_jar_path,
-               ivy_settings=self._ivy_subsystem.get_options().ivy_settings,
                ivy_cache_dir=self._ivy_subsystem.get_options().cache_dir,
                extra_jvm_options=self._ivy_subsystem.extra_jvm_options())
